### PR TITLE
Add IRC channel badge to README.md.

### PR DESCRIPTION
**Pull request by jankeromnes**

### CODE REVIEW

**Comment by jankeromnes** [comment]
Idea stolen from https://github.com/Chocobozzz/PeerTube/blob/develop/README.md

File: README.md [hunk 1]
```diff
@@ -4,12 +4,13 @@
 [![Docker Hub](https://img.shields.io/docker/build/janitortechnology/janitor.svg)](https://hub.docker.com/r/janitortechnology/janitor/)
 [![Greenkeeper](https://img.shields.io/badge/greenkeeper-enabled-brightgreen.svg)](https://greenkeeper.io/)
 [![NPM version](https://img.shields.io/npm/v/janitor.technology.svg)](https://www.npmjs.com/package/janitor.technology)
+[![IRC channel](https://img.shields.io/badge/%23janitor-on%20freenode-brightgreen.svg)](https://kiwiirc.com/client/irc.freenode.net/?#janitor "irc.freenode.net#janitor")
```
Comment (nt1m) [inline README.md]: newclient instead of client ?

Comment (jankeromnes) [reply README.md]: I like the idea, but I get a 404 on https://kiwiirc.com/newclient/irc.freenode.net/?#janitor

Won't they enable the new client by default once it's ready?

Comment (nt1m) [reply README.md]: I don't mind either way :)

**Pull request merged**